pytest_fixtures/broker.py

```diff
@@ -39,3 +41,17 @@ def rhel77_contenthost_class(request):
     with VMBroker(host_classes={'host': ContentHost}, **BROKER_RHEL77) as host:
         request.cls.content_host = host
         yield
+
+
+@pytest.fixture
+def satellite_latest():
```

Comment:
This is problematic, as the test session may not be running against a satellite that matches the version that will be resolved by a broker call that doesn't specify a version.  Even if this isn't backported to the 6.8.z branch, it will become a problem with 6.9.z branch is created.

Reply:
For the time being, this is safe (works in non-Z runs). But we can revisit after dynaconf phase 2.